molecule/driver/openstackdriver.py

```diff
@@ -22,6 +22,9 @@
 import os
 
 import shade
+import paramiko
+import time
+import tempfile
```

Comment:
Since time and tempfile are part of the standard library, they should be imorted with OS, in alphabetical order.  Paramiki should stay with shade, however, should go before it.